terrible.py

```diff
@@ -0,0 +1,17 @@
+#!/usr/bin/env python3
```

Comment:
This file contains unused source code.

*Origin: PyUnusedCodeBear, Section: `all.python`.*

The issue can be fixed by applying the following patch:

```diff
--- a/tmp/tmpcbzce6lr/terrible.py
+++ b/tmp/tmpcbzce6lr/terrible.py
@@ -1,7 +1,4 @@
 #!/usr/bin/env python3
-import sys
-import os
-import time
 name='Yuki'
 class Greeter:
     def greet(self):
```